src/main/java/edu/greenblitz/robotName/commands/arm/roller/ShootFromRollerToAmp.java

```diff
@@ -0,0 +1,26 @@
+package edu.greenblitz.robotName.commands.arm.roller;
+
+import edu.greenblitz.robotName.subsystems.arm.EndEffector.RollerUtils.RollerConstants;
+import edu.greenblitz.robotName.subsystems.arm.Wrist;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class ShootFromRollerToAmp extends RollerCommand{
+    @Override
+    public void initialize() {
+        roller.rollIn();
+    }
+
+    @Override
+    public boolean isFinished() {
+        if (!Wrist.getInstance().isObjectInside()){
+            new WaitCommand(RollerConstants.TIME_AFTER_OBJECT_IN_ROBOT);
```

Comment:
This wouldn't work, you are creating a new instance of a command, you need to schedule it and wait for it to finish. It should be simpler to simply check for the time in the isFinished function.